onnx/examples/make_model.ipynb

```diff
@@ -3,11 +3,55 @@
   {
    "cell_type": "code",
    "execution_count": 1,
-   "metadata": {},
+   "source": [
```

Comment:
Changes to this file dont look relevant

Reply:
Actually I only change this line `https://github.com/onnx/onnx/blob/master/onnx/onnx.proto` into `https://github.com/onnx/onnx/blob/main/onnx/onnx.proto`. My vscode somehow changed the key order here and make the change look irrelevant... Will try to make it look more reasonable.